app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
     <activity android:name=".player.state.testing.StateFragmentTestActivity" />
     <activity android:name=".player.exploration.ExplorationActivity" />
     <activity android:name=".topic.questionplayer.QuestionPlayerActivity" />
-    <activity android:name=".topic.TopicActivity" />
+    <activity android:name=".topic.TopicActivity"
```

Comment:
Please maintain alphabetical order for activities.

Reply:
Will be done in separate PR